add-serve-target.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
See above - script adds a "serve" target to the generated nx next app used in the plugin test